src/libraries/System.Runtime.InteropServices/src/System/Runtime/InteropServices/Marshalling/StrategyBasedComWrappers.cs

```diff
@@ -8,17 +8,40 @@
 
 namespace System.Runtime.InteropServices.Marshalling
 {
+    /// <summary>
+    /// A <see cref="ComWrappers"/>-based type that uses customizable strategy objects to implement COM object wrappers and managed object wrappers exposed to COM.
+    /// </summary>
     [CLSCompliant(false)]
     public class StrategyBasedComWrappers : ComWrappers
```

Comment:
Should we mention anything about how this works with the GeneratedComInterface interfaces by default?

Reply:
Good idea, I'll adjust the wording on the default strategies docs to say it works with source-generated COM.